src/openms/include/OpenMS/METADATA/PeptideHit.h

```diff
@@ -84,6 +84,24 @@ namespace OpenMS
     };
     //@}
 
+    /// Analysis Result (containing search engine / prophet results)
+    class OPENMS_DLLAPI AnalysisResult
```

Comment:
well, in pepxml this is called analysis_result so I think the name makes sense, leading to less confusion than having 2 names for the same thing